tmt/steps/prepare/ansible.py

```diff
@@ -93,13 +108,12 @@ class PrepareAnsible(tmt.steps.prepare.PreparePlugin[PrepareAnsibleData]):
         prepare:
             how: ansible
             playbook:
-              - playbook/one.yml
               - https://foo.bar/two.yml
-              - playbook/three.yml
+              - two.yml
 
-    The playbook path must be relative to the metadata tree root.
+    .. code-block:: shell
 
-    The playbook path should be relative to the metadata tree root.
+        prepare -h ansible --playbook https://foo.bar/two.yml --playbook two.yml
```

Comment:
```suggestion
        prepare --how ansible --playbook https://foo.bar/two.yml --playbook two.yml
```

What about rather using long option variants in the examples? To make it obvious to the user at the first sight what the option is about.